packages/sveltekit/package.json

```diff
@@ -9,7 +9,8 @@
     "types": "./lib/index.d.ts",
     "import": "./lib/index.js",
     "require": "./lib/index.js",
-    "default": "./lib/index.js"
+    "default": "./lib/index.js",
+    "svelte": "./lib/index.js"
```

Comment:
nit: Need to add a changeset and release a new patch

Reply:
changeset added